SpecflowPages/Helpers/ExtentReportManager.cs

```diff
@@ -0,0 +1,39 @@
+﻿using AventStack.ExtentReports;
+using AventStack.ExtentReports.Reporter;
+using MarsQA.Helpers;
+
+namespace MarsQA.SpecflowPages.Helpers
+{
+    public class ExtentReportManager
```

Comment:
why did you implement extend reporter?

Reply:
In my base code there was object defined ExtentTest & ExtentReports in CommonMethods class, so Learning purpose I have added it